app/src/main/java/com/kickstarter/viewmodels/ConsentManagementDialogFragmentViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package com.kickstarter.viewmodels
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.kickstarter.libs.Environment
+import com.kickstarter.ui.SharedPreferenceKey
+import rx.subjects.BehaviorSubject
```

Comment:
You are still using rx1 observables, if you enherit from `ViewModel()` and not from `BaseFragment` there is no disposition of subscriptions.
The rxjava2 observables are imported from `io.reactivex.subjects.BehaviorSubject`